misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/AbstractCharSequenceExpectationsTest.kt

```diff
@@ -33,6 +37,24 @@ abstract class AbstractCharSequenceExpectationsTest(
         notToMatchSpec.forSubjectLessTest(Regex(""))
```

Comment:
please also add the entries here:
```suggestion
        notToMatchSpec.forSubjectLessTest(Regex("")),
        lengthFeatureSpec.forSubjectLessTest(),
        lengthSpec.forSubjectLessTest{ toBeLessThan(1) },
```